clientconn.go

```diff
@@ -411,6 +433,9 @@ func (cc *ClientConn) Close() error {
 	}
 	cc.state = Shutdown
 	cc.stateCV.Broadcast()
+	if cc.events != nil {
+		cc.events.Finish()
```

Comment:
is there any problem if cc.event.Errorf/Printf get called after Finish is called?